docs/components/camera/single-stream.md

```diff
@@ -121,7 +121,7 @@ The following attributes are available for `single_stream` cameras:
 | `distortion_parameters` | object | Optional | Modified Brown-Conrady parameters used to correct for distortions caused by the shape of the camera lens: <ul> <li> <code>rk1</code>: The radial distortion x. </li> <li> <code>rk2</code>: The radial distortion y. </li> <li> <code>rk3</code>: The radial distortion z. </li> <li> <code>tp1</code>: The tangential distortion x. </li> <li> <code>tp2</code>: The tangential distortion y. </li> </ul> |
 | `debug` | boolean | Optional | Enables the debug outputs from the camera if `true`. <br> Default: `false` |
 
-If you have a camera that uses its own SDK to access its images and point clouds (like an [Intel RealSense](https://app.viam.com/module/viam/realsense) or a [Luxonis OAK-D](https://app.viam.com/module/viam/oak-d)), you can add a camera server as a {{< glossary_tooltip term_id="remote" text="remote" >}} component of your machine.
+If you have a camera that uses its own SDK to access its images and point clouds (like an [Intel RealSense](https://app.viam.com/module/viam/realsense) or a [Luxonis OAK-D](https://app.viam.com/module/viam/oak-d)), you can add a camera server as a {{< glossary_tooltip term_id="remote-part" text="remote" >}} component of your machine.
```

Comment:
Yeah I noticed this as well. I'm also not sure what they mean by "camera server"--does that make sense to you? I'm wondering if this is referring to the old, pre-modular registry way of doing it https://docs.viam.com/registry/advanced/custom-components-remotes/
If we're not sure, I think it's okay to leave as-is in this PR other than changing the glossary ID since this is out of diff?

Reply:
Sounds good!